packages/react-hooks/src/ConnectionProvider.tsx

```diff
@@ -15,21 +15,57 @@ import {
   addRecord,
 } from './recordUtils'
 
+export type useConnectionsOptions = {
+  excludedTypes?: [ConnectionType | string]
```

Comment:
`[ConnectionType | string]` means an array of length one with ConnectionType/string. While `Array<ConnectionType | string>` means an array with ConnectionType/string

```suggestion
  excludedTypes?: Array<ConnectionType | string>
```